pyproject.toml

```
@@ -0,0 +1,199 @@
+[build-system]
+requires = ['setuptools', 'wheel']
+build-backend = 'setuptools.build_meta'
+
+[project]
+authors = [{ name = 'DataCebo, Inc.', email = 'info@sdv.dev' }]
+classifiers = [
```

keep this at the top
```suggestion
name = 'sdgym'
description = 'Benchmark tabular synthetic data generators using a variety of datasets'
authors = [{ name = 'DataCebo, Inc.', email = 'info@sdv.dev' }]
classifiers = [
```

---

Yes, done in [7fac59d](https://github.com/sdv-dev/SDGym/pull/269/commits/7fac59d7ad872f32dc3b7307944cfe1980724f17)